src/index.js

```diff
@@ -0,0 +1,45 @@
+import navBar from './navbar';
+import homeMainPage from './mainPage';
+import menu from './menu';
+import contactPage from './contact';
+
+// navBar();
+// homeMainPage();
+// // menu();
```

Comment:
- [ ] Since at this stage we are about to merge our code to the production stage, it makes sense to delete all commented code.